tests/regenerate_models.sh

```diff
@@ -16,23 +16,23 @@ rm -rf test_cli_classification_mol
 
 chemprop train -i $CHEMPROP_PATH/tests/data/classification/mol.csv --accelerator cpu --epochs 1 --num-workers 0 --task-type classification --save-dir test_cli_classification_mol
 
-cp -L test_cli_classification_mol/model_0/model.pt $CHEMPROP_PATH/tests/data/example_model_v2_classification_mol.pt
+cp -L test_cli_classification_mol/model_0/best.pt $CHEMPROP_PATH/tests/data/example_model_v2_classification_mol.pt
 
 # test_cli_classification_mol_multiclass
 
 rm -rf test_cli_classification_mol_multiclass
 
 chemprop train -i $CHEMPROP_PATH/tests/data/classification/mol_multiclass.csv --accelerator cpu --epochs 1 --num-workers 0 --save-dir test_cli_classification_mol_multiclass --task-type multiclass
 
-cp -L test_cli_classification_mol_multiclass/model_0/model.pt $CHEMPROP_PATH/tests/data/example_model_v2_classification_mol_multiclass.pt
+cp -L test_cli_classification_mol_multiclass/model_0/best.pt $CHEMPROP_PATH/tests/data/example_model_v2_classification_mol_multiclass.pt
 
 # test_cli_regression_mol+mol
 
 rm -rf test_cli_regression_mol+mol
 
 chemprop train -i $CHEMPROP_PATH/tests/data/regression/mol+mol/mol+mol.csv --accelerator cpu --epochs 1 --num-workers 0 --smiles-columns smiles solvent --save-dir test_cli_regression_mol+mol
 
-cp -L test_cli_regression_mol+mol/model_0/model.pt $CHEMPROP_PATH/tests/data/example_model_v2_regression_mol+mol.pt
+cp -L test_cli_regression_mol+mol/model_0/best.pt $CHEMPROP_PATH/tests/data/example_model_v2_regression_mol+mol.pt
 
 cp -L test_cli_regression_mol+mol/model_0/checkpoints/last.ckpt $CHEMPROP_PATH/tests/data/example_model_v2_regression_mol+mol.ckpt
```

Comment:
Change `last.ckpt` to the `best-{epoch}-{val_loss:.2f}.ckpt`?

Reply:
What is hard about that is the name is dynamically made by lightning. We could use `.../model_0/checkpoint/best*.ckpt`. Or we could use `last.pt` and `last.ckpt` as these are trained for 1 epoch anyways. But if we want to not save `last.pt` (fine by me), then using a glob (`*`) is probably the best route. We have some tests where we check that the .pt and .ckpt give the same prediction, so we need to be consistent.